launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/servlets/TranslationServlet.java

```diff
@@ -0,0 +1,108 @@
+package com.google.launchpod.servlets;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import com.google.cloud.translate.Translate.TranslateOption;
+import com.google.launchpod.data.Item;
+import com.google.launchpod.data.RSS;
+
+@WebServlet("/translate-feed")
+public class TranslationServlet extends HttpServlet {
+
+  private static final long serialVersionUID = 1L;
+  private static final String USER_FEED = "UserFeed";
+  private static final String RSS_FEED_LINK = "rssFeedLink";
+  private static final String LANGUAGE = "language";
+  private static final String XML_STRING = "xmlString";
+  private static final XmlMapper XML_MAPPER = new XmlMapper();
+  private static final String BASE_URL = "https://launchpod-step18-2020.appspot.com/rss-feed?id=";
+
+  @Override
+  public void doPost(HttpServletRequest req, HttpServletResponse res)
+      throws JsonParseException, JsonMappingException, IOException {
+    String link = req.getParameter(RSS_FEED_LINK);
+    String targetLanguage = req.getParameter(LANGUAGE);
+    if (link == null || link == "") {
+      throw new IOException("Please give valid link.");
+    }
+    if (targetLanguage == null || targetLanguage == "") {
+      throw new IOException("Please give valid language.");
+    }
+    try {
+      XML_MAPPER.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```

Comment:
you can add this in a static initialization block right after you define this field:

static {
      XML_MAPPER.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
}